### PR TITLE
Lightly server https

### PR DESCRIPTION
**Pull request by japrescott**
closes lig-5082
- allow to pass ssl certs to lightly-serve to start it in HTTPS mode

### CODE REVIEW

File: lightly/cli/serve_cli.py [hunk 1]
```diff
@@ -19,9 +20,13 @@ def lightly_serve(cfg):
         lightly_mount:
             Path to the Lightly directory.
         host:
-            Hostname for serving the data (defaults to localhost).
+            Hostname for serving the data (defaults to localhost). If you want to expose this to the world, use '0.0.0.0'.
```
Comment (MalteEbner) [inline lightly/cli/serve_cli.py:23]: [Similar to Ersi](https://lightly-ai.slack.com/archives/C01KSD8EE65/p1717435699756949?thread_ts=1717432537.232419&cid=C01KSD8EE65), I don't get what "exposing to the world" means. When would I use this more specifically? Exposing to the world sounds very bad from a security perspective, when is it ok?
```suggestion
            Hostname for serving the data (defaults to localhost). Set it to '0.0.0.0' to ...
```

**Review by MalteEbner — APPROVED** [review]
Apart from the docstring, I approve.

File: lightly/cli/serve_cli.py (hunk 1 above)
Comment (japrescott) [reply lightly/cli/serve_cli.py:23]: It **is** "bad" from a security perspective if you don't have any other security measures in place, yes. E.g if you allow other people to access the same network as you or you are in a public WLAN, other people **will** be able to see your server.

Comment (japrescott) [reply lightly/cli/serve_cli.py]: I changed it to `If you want to expose it to the internet or your local network, use '0.0.0.0'.`. Does that explain it better to you Malte?

Comment (MalteEbner) [reply lightly/cli/serve_cli.py]: Alternatively, don't explain it, but link to the new docs.

**Pull request merged**